cyences_app_for_splunk/bin/cyences_send_email.py

```diff
@@ -14,6 +14,8 @@
 
 
 ALERT_ACTION_NAME = 'cyences_send_email_action'
+SOC_TEAM_EMAIL_MACRO = 'cs_soc_team_email'
```

Comment:
Are we storing email addresses in this macro?

Reply:
yes